feature-libs/pickup-in-store/core/store/actions/pickup-location.action.ts

```diff
@@ -81,7 +82,30 @@ export const SetStoreDetailsSuccess = createAction(
   props<{ payload: PointOfService }>()
 );
 
-export const SetStoreDetailsFailure = createAction(
-  STORE_DETAILS_FAIL,
-  props<{ error: ErrorActionType }>()
-);
+interface TypeOf_SetStoreDetailsFailure {
```

Comment:
Good point, `TypeOf_` it looks odd even to me. (... "odd even" 🙃).
It would be too bad if anyone gets inspired by this and copies it to other places.

I'll try your suggestion.